README.md

```diff
@@ -48,7 +48,7 @@ $ ruby-build --definitions             # lists all available versions of Ruby
 $ ruby-build 2.2.0 ~/local/ruby-2.2.0  # installs Ruby 2.2.0 to ~/local/ruby-2.2.0
 ```
 
-> **Warning**  
```

Comment:
The whitespace here is meaningful. Please don't commit unrelated changes that your editor makes. Thanks!

Reply:
Oops, I was so careless. Retrieved those spaces. [ad71af3](https://github.com/rbenv/ruby-build/pull/2151/commits/ad71af30d0ab8b61d7a13da28af42873de497674)